scripts/run.sh

```diff
@@ -36,8 +46,9 @@ fi
 
 # Install everything, including any new dependencies
 pip uninstall dask distributed
-pip install -e ./dask[test]
-pip install -e ./distributed
+pip install pandas[test] pytest pytest-cov pytest-mock pytest-rerunfailures pytest-timeout pytest-xdist pre-commit
+pip install --no-deps -e ./dask
+pip install --no-deps -e ./distributed
```

Comment:
I'd like to be able to pin to a specific version of `dask` in `rapids-dask-dependency` without "breaking" the `dask:main` installation. When building locally, I simply use `--no-deps`. However, I assume we **do** need to somehow ensure the testing deps are satisfied?

Reply:
What happens without `--no-deps`? I'd expect to maybe get a warning saying that the version conflicts with what's pinned in `rapids-dask-dependency`, but things should still go through.

Reply:
Hmm - My worry is that pip would try to install older versions of dask-cudf/dask-cuda and would ultimately raise an error if it couldn't.